### PR TITLE
Clarify the usage of crash flip arrow

### PR DESCRIPTION
**Pull request by atomgomba**
Probably it's just me, but it has been puzzling me for a while why I cannot get crash flip arrow to display despite it's enabled on the OSD.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
